ql/cashflows/averagebmacoupon.cpp

```diff
@@ -100,6 +100,12 @@ namespace QuantLib {
 
     }
 
+    namespace {
+    void adjustToNextValidFixingDate(Date& d, const ext::shared_ptr<BMAIndex>& index) {
```

Comment:
"Previous", since we're going backwards?  I'm not sure.

Reply:
that's better, yes, I'll change it